quadlet/valkey/valkey.container

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=A Valkey development container
+Wants=network-online.target
+After=network-online.target
+
+[Container]
+Image=docker.io/valkey/valkey:7.2
```

Comment:
Valkey does not have a `:latest` or `:stable` tag.  
The only tag that could be set and forgotten is the `:unstable` tag, but I thought this would not be wanted.

Reply:
I don't see a problem. But, maybe add a comment about it in the README file

Reply:
Added the following text to the README, and squashed the commits into a single one

> Valkey does not currently publish a `:latest` or `:stable` tag, so the quadlet contains the hard-coded version `7.2`. If you would like to use a different version, you can modify the image tag inside the `valkey.container` file.